### PR TITLE
Add docs on browser OTeL

### PR DESCRIPTION
**Pull request by ccschmitz**
## Summary

* Adds a docs page on our OTeL browser instrumentation
* Adds a blog post on browser OTeL
* Updates our fullstack mapping docs to mention using context propogation as a means of connecting resources across a distributed system
* Fixes rendering of lists on the blog (was rendering an `ol` or `ul` for each list item) and tweaks styling of lists

Screenshot 2024-08-23 at 10 27 07 AM

Screenshot 2024-08-23 at 4 48 23 PM

## How did you test this change?

Click tested a local build of the site.

## Are there any deployment considerations?

N/A - only a docs change.

## Does this work require review from our design team?

N/A

### CODE REVIEW

File: docs-content/getting-started/3_client-sdk/7_replay-configuration/opentelemetry.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+4. Fetch Requests: Similar to XHR, this captures information about outgoing fetch requests, including URL, method, and timing data.
+
+5. GraphQL Operations: For applications using GraphQL, the SDK can capture and parse GraphQL operations, providing insights into query and mutation performance.
+
+These auto-instrumentations provide a comprehensive view of your application's performance from the client-side perspective, allowing you to identify bottlenecks, track user interactions, and monitor network requests without manual instrumentation.
+
+## Connecting Client and Server Traces
+
+One of the powerful features of OpenTelemetry is the ability to connect traces across different services and environments, including from the client to the server. Highlight.io's SDK facilitates this connection through context propagation. This allows you to create end-to-end traces that span from user interactions in the browser all the way to your backend services.
```
Comment (jay-khatri) [inline docs-content/getting-started/3_client-sdk/7_replay-configuration/opentelemetry.md:41]: is there somewhere we should link to this?

Comment (ccschmitz) [reply docs-content/getting-started/3_client-sdk/7_replay-configuration/opentelemetry.md:41]: Are you thinking we should link out to OTeL docs on context propogation?

File: blog-content/monitoring-browser-applications-with-opentelemetry.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+Here's how it works:
+
+1. **Trace Initiation**: When a user interaction or a network request is initiated in the browser, the SDK creates a new span or continues an existing trace.
+2. **Header Injection**: For outgoing HTTP requests (both XHR and Fetch), the SDK automatically injects trace context headers into the request. These headers typically include:
+   - `traceparent`: Contains the trace ID, parent span ID, and trace flags.
+   - `tracestate`: Carries vendor-specific trace information.
+3. **Server-Side Reception**: The Highlight server-side SDKs extract these headers and continue the trace, linking the server-side spans to the client-side trace.
+4. **Trace Completion**: As the request completes and returns to the client, the full trace, including both client and server spans, can be visualized in Highlight's UI.
+
+This connection between client and server traces provides end-to-end visibility, facilitates performance optimization, and enables error correlation.
```
Comment (Vadman97) [inline blog-content/monitoring-browser-applications-with-opentelemetry.md:65]: could be good to add the server -> client propagation here as well. that could link to the `Handling Server-Initiated Traces` section below?

For server-side rendering where code executes server side before HTML is sent to the browser, the trace context is propagated through a meta tag that is added to the HTML.

File: docs-content/getting-started/3_client-sdk/7_replay-configuration/opentelemetry.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## Enabling OpenTelemetry with Highlight
+
+To enable OpenTelemetry tracing in your application using Highlight's SDK, set the `enableOtelTracing` configuration option when initializing the SDK:
+
+```ts
+H.init({
+  // ...
+  enableOtelTracing: true
+})
```
Comment (Vadman97) [inline docs-content/getting-started/3_client-sdk/7_replay-configuration/opentelemetry.md:18]: should we update the sdk docs on this config? or are we keeping that for another pr

File: blog-content/monitoring-browser-applications-with-opentelemetry.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+
+These insights allow developers to identify bottlenecks, track user interactions, and monitor network requests without manual instrumentation. What's mentioned above is data you get relatively easily by leveraging [OTeL's auto instrumentations](https://opentelemetry.io/docs/zero-code/js/) but you can also add manual instrumentation to collect traces on any other code in you client-side apps.
+
+## Highlight and OpenTelemetry
+
+Highlight's JavaScript SDK offers built-in support for collecting OpenTelemetry data from client-side applications. This integration allows you to seamlessly incorporate OpenTelemetry tracing into your web applications.
+
+### Enabling OpenTelemetry with Highlight
+
+OTeL data collection is still in beta, so you need to explicitly enable it by setting the `enableOtelTracing` configuration option when initializing the SDK:
```
Comment (Vadman97) [inline blog-content/monitoring-browser-applications-with-opentelemetry.md:41]: are we ready to make this the default rather than opt in?

**Review by Vadman97 — APPROVED** [review]

**Review by jay-khatri — APPROVED** [review]
lgtm. my only concern is that it may be hard for people to find the browser otel doc? Should we embed this in each of our frontend SDK tutorials here: https://www.highlight.io/docs/getting-started/overview#for-your-frontend?

Even if we're going to make this opt-out soon, it probably doesn't hurt to just add it as an `h.init` parameter with a comment that links to the doc?

File: docs-content/getting-started/3_client-sdk/7_replay-configuration/opentelemetry.md (hunk 3 above)
Comment (ccschmitz) [reply docs-content/getting-started/3_client-sdk/7_replay-configuration/opentelemetry.md:18]: Yeah, probably a good idea to advertise that at this point!

**Comment by ccschmitz** [comment]
@jay-khatri yeah, I think you're right. I'll update the API docs and the tutorials to mention this as well.

**Comment by ccschmitz** [comment]
> lgtm. my only concern is that it may be hard for people to find the browser otel doc? Should we embed this in each of our frontend SDK tutorials here: https://www.highlight.io/docs/getting-started/overview#for-your-frontend?

Looked at and thought about this a bit more and I don't think it makes sense to add this to the docs. I added it to the SDK docs, and I think I'll add a section in the fullstack mapping doc about connecting server-initiated traces.

**Pull request merged**